source/Jobbr.WebApi.Tests/ExposeStorageProvider.cs

```diff
@@ -1,17 +1,17 @@
-using Jobbr.ComponentModel.JobStorage;
+using Jobbr.ComponentModel.JobStorage;  
 using Jobbr.ComponentModel.Registration;
 
 namespace Jobbr.WebApi.Tests
 {
     public class ExposeStorageProvider : IJobbrComponent
     {
-        internal readonly IJobStorageProvider JobStorageProvider;
+        internal IJobStorageProvider JobStorageProvider { get; }

```

Comment:
Why use a property?

Reply:
it is accessed like ExposeStorageProvider.Instance.JobStorageProvider in the setup for the integration tests